src/pages/Chat.js

```diff
@@ -54,14 +54,14 @@ function Header({ chatState }) {
 
 export default function Chat() {
   const { currentChatToken, isError: tokenError } = useCurrentChatToken();
-  const { currentPatient, isError: patientError } = useCurrentPatient();
+  const { currentPatient } = useContext(CurrentUserContext);
 
   const [connectionState, setConnectionState] = useState("disconnected");
   const [conversation, setConversation] = useState(null);
   const [requestError, setRequestError] = useState(false);
   const [noExistingConv, setNoExistingConv] = useState(false);
 
-  if (tokenError || patientError) setRequestError(true);
+  if (tokenError) setRequestError(true);
```

Comment:
Same as previous comment about using brackets
```suggestion
  if (tokenError) {
    setRequestError(true);
  }
```